data/blog/web-preferences-fix.md

```diff
@@ -0,0 +1,66 @@
+---
+title:  WebPreferences Vulnerability Fix
+author: ckerr
+date: '2018-08-22'
+---
+
+A remote code execution vulnerability has been discovered affecting  apps with the ability to open consecutive child windows on Electron versions (3.0.0-beta.6, 2.0.7, 1.8.7, and 1.7.15) for. This vulnerability has been assigned the CVE identifier [CVE-2018-15685].
```

Comment:
s/consecutive/nested

(hierarchy is the important thing here)

Reply:
The wording of "with the ability to open" is ambiguous -- does the CVE have this ability or does it affect apps that have this ability?